### PR TITLE
Set a specific formatting specifier for rl.rlim_max.

### PR DESCRIPTION
**Pull request by katre**
Part of error message cleanup, #6594.

### CODE REVIEW

**Comment by katre** [comment]
The error message in question:
```
INFO: From Compiling src/test/cpp/blaze_util_posix_test.cc:
src/test/cpp/blaze_util_posix_test.cc:192:52: warning: format specifies type 'uintmax_t' (aka 'unsigned long') but the argument has type 'rlim_t' (aka 'unsigned long long') [-Wformat]
            "; cannot test anything meaningful\n", rl.rlim_max);
                                                   ^~~~~~~~~~~
1 warning generated.
```

Please let me know if there's a better way to fix this.

**Comment by jmmv** [comment]
The addition of the cast is the right thing to do, but the removal of `PRIuMAX` is not. Doesn't it work if you leave the macro in place?

**Comment by katre** [comment]
I thought it didn't, but on re-testing it does still work with PRIuMAX, so added it back in.

**Comment by jmmv** [comment]
Please reintroduce the inclusion of `stdint.h` to bring `uintmax_t` into scope. Or better, include `cstdint` which is the C++ counterpart.

Also, while at it, I'd take the chance to switch to a C++ style cast per the style guide: https://google.github.io/styleguide/cppguide.html#Casting ; i.e. `static_cast<uintmax_t>(foo)`.

File: src/test/cpp/blaze_util_posix_test.cc [hunk 1]
```diff
@@ -16,14 +16,14 @@
...
 
-#include <inttypes.h>
 #include <string.h>
 #include <unistd.h>
 
 #include <cerrno>
 #include <cstdarg>
 #include <cstdio>
 #include <cstdlib>
+#include <cstdint>
```
Comment (jmmv) [inline src/test/cpp/blaze_util_posix_test.cc]: Please keep includes sorted alphabetically. This has to come before cstdio.

**Comment by jmmv** [comment]
When you squash all these commits, please make sure that the description reflects the actual change that was finally done.

**Comment by jmmv** [comment]
Looks good with the reordering mentioned above though.

File: src/test/cpp/blaze_util_posix_test.cc (hunk 1 above)
Comment (katre) [reply src/test/cpp/blaze_util_posix_test.cc]: Done.

**Pull request closed without merging**